src/scripts/projectCreation.ts

```diff
@@ -0,0 +1,89 @@
+import { Toolbox } from 'gluegun/build/types/domain/toolbox';
+import { interfaceHelpers } from '../utils/interface';
+import { AIRSHIP_EMAIL, DEFAULT_PROJECT_VERSION, RepoLocation } from '../constants';
+const decamelize = require('decamelize');
+
+/**
+ * Clones the template from the repository on Airship's GitHub profile
+ * @param repo location of the template repository
+ * @param projectName name of the project
+ * @param toolbox Gluegun-supplied toolbox
+ */
+export const cloneTemplateRepo = async (
+  projectName: string,
+  repo: RepoLocation,
+  toolbox: Toolbox,
+) => {
+  const { cmd, printTask } = interfaceHelpers(toolbox);
+  const task = printTask('☀️  Opening hangar door...');
+  await cmd(`git clone ${repo} ${projectName}`);
+  task.stop();
+};
+
+/**
+ * Initializes the git repository for the project. It does the following:
+ * 1. Deletes the git history from the template repo
+ * 2. Re-initializes git for the project
+ * 3. Makes an initial commit
+ * 4. Renames the "master" branch to "main"
```

Comment:
I wonder if we can nix these comments, since the commands below are easy to read.

One issue with code comments is that they double the amount of things that need to be updated when changes are made.